docker-compose.yml

```diff
@@ -95,22 +92,11 @@ services:
     command: node run.js
```

Comment:
This should be
```suggestion
     command: node index.js
```

Reply:
The KS patch originally had index.js, but I changed it back to run.js so that I wouldn't have to modify it here or in the Dockerfile.  So I'm pretty sure this is correct as it stands.

Reply:
Yep, I was a few commits behind, on e3f80902fe52895b2463ab4e5f9ee2726b74d678.